ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/metacard-actions/metacard-actions.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import { hot } from 'react-hot-loader'
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+import MapActions from '../../container/map-actions'
+
+type Props = {
+  model: Backbone.Model
+  exportActions: any
+  otherActions: any
+}
+
+const Root = styled.div`
+  overflow: auto;
+  height: 100%;
+  padding: 0px ${props => props.theme.largeSpacing};
+
+  .is-header {
```

Comment:
Each of these class selectors should be their own styled component. We can remove the classes from the JSX elements as well.